installation/install_cross_deps_ipks.bash

```diff
@@ -6,101 +6,82 @@
-	tar xzf data.tar.gz -C /usr/arm-frc-linux-gnueabi
-	cd
-	rm -rf .frc_control_temp
+    mkdir ~/.frc_control_temp
+    cd ~/.frc_control_temp
+    wget -q --show-progress "$1"
+    ar x "${1##*/}" # Note: This regex simply extracts the filename from the url
+    tar xzf data.tar.gz -C ~/frc2019/roborio/arm-frc2019-linux-gnueabi
+    cd
+    rm -rf .frc_control_temp
 }
 
-prefix="http://download.ni.com/ni-linux-rt/feeds/2017/arm/cortexa9-vfpv3/"
-
-## Install Boost
-install_package "${prefix}boost-dev_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-# install_package "${prefix}boost-log_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-# install_package "${prefix}boost-serialization_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-# install_package "${prefix}boost-staticdev_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-# install_package "${prefix}boost-test_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-# install_package "${prefix}boost_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-atomic1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-chrono1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-date-time1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-filesystem1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-graph1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-iostreams1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-program-options1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-random1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-regex1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-signals1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-system1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-thread1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-install_package "${prefix}libboost-timer1.60.0_1.60.0-r0.3_cortexa9-vfpv3.ipk"
-
-## Install Python
-install_package "${prefix}python-core_2.7.11-r1.49_cortexa9-vfpv3.ipk"
-install_package "${prefix}libpython2_2.7.11-r1.49_cortexa9-vfpv3.ipk" # Required for python_orocos_kdl
-install_package "${prefix}python-dev_2.7.11-r1.49_cortexa9-vfpv3.ipk" # Required for python_orocos_kdl
-
-## Install Eigen (Required for various packages (eigen, orocos, kdl, etc))
-install_package "${prefix}libeigen_3.2.6-r0.5_cortexa9-vfpv3.ipk"
+prefix="http://download.ni.com/ni-linux-rt/feeds/2018.5/arm/cortexa9-vfpv3/"
+
+# Install Boost
+install_package "${prefix}boost-dev_1.63.0-r1.5_cortexa9-vfpv3.ipk"
+# install_package "${prefix}boost-log_1.63.0-r1.5_cortexa9-vfpv3.ipk"
+# install_package "${prefix}boost-serialization_1.63.0-r1.5_cortexa9-vfpv3.ipk"
+# install_package "${prefix}boost-staticdev_1.63.0-r1.5_cortexa9-vfpv3.ipk"
+# install_package "${prefix}boost-test_1.63.0-r1.5_cortexa9-vfpv3.ipk"
+# install_package "${prefix}boost_1.63.0-r1.5_cortexa9-vfpv3.ipk"
```

Comment:
Will these boost library's ever be used again? Is there still a reason to have them commented out here?